### ISSUE TITLE
[Service] Problem with INGV dataselect

### ISSUE THREAD

**Issue by javiquinte** [EIDA/userfeedback#15]
Connections to the application.wadl method of the Dataselect WS from INGV stop after 60 seconds with a timeout.

This has been tried with an fdsnws_fetch command which is used almost every day to check that the Alparray data can be delivered.

`fdsnws_fetch -vvv -N Z3 -C "HHZ" -s "2016-03-01" -e "2016-03-01T00:02:00" -o data.mseed`

The error looks like:

> retrying http://webservices.ingv.it/fdsnws/dataselect/1/application.wadl (2) after 60 seconds due to <urlopen error [Errno -3] Temporary failure in name resolution>

**Comment by flofux** [comment]
I also cannot access INGV services today.

```
from obspy.clients.fdsn import Client
client = Client('INGV')
```
FDSNException: No FDSN services could be discovered at 'http://webservices.ingv.it'. This could be due to a temporary service outage or an invalid FDSN service address.

**Comment by massimo1962** [comment]
Hi, the problems are due to general network issues (DNS, IP Routing, etc..) and not to the services itself.
our network admins are working on this, we are supporting them as much as we can. However, we (the datacentre) cannot solve these problems.

We already confirmed missing DNS records in parts of the internet. Is this the same issue you are seeing?

**Comment by petrrr** [comment]
The DNS issues should have been solved.
Could you give us a feedback and close if solved.?

**Comment by javiquinte** [comment]
Dataselect service seems to be up again, but is not working properly. DNS issue seems to be solved.
#16

**Comment by massimo1962** [comment]
here we can say that DNS issue it's solved, I close this issue and continue in #16.
